contrib/build_rpm.sh

```diff
@@ -60,6 +67,3 @@ if [ -d ~/rpmbuild/BUILD ]; then
 fi
 
 rpmbuild --rebuild ${extra_arg:-} podman-*.src.rpm
-
-# clean up src.rpm as it's been built
-sudo rm -f podman-*.src.rpm
```

Comment:
This is a significant change in behavior and isn't really explained.

Reply:
In fact I'm uncomfortable with the `rm` being done in Makefile; what would you think of moving this up a few lines, to just before the `make -f .copr/Makefile`?